### PR TITLE
Main article

### LINKED ISSUE #46: Component: Main / Article
Article component.

### PR DESCRIPTION
**Pull request by TeddTech**
resolves #46

### CODE REVIEW

File: components/Article.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<template>
+	<div>
```
Comment (greatislander) [inline components/Article.vue]: For an article, can you suggest a more semantically appropriate container to use?

File: components/Article.vue [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		<h1 class="title">
+			<b>{{ Title }}</b>
```
Comment (greatislander) [inline components/Article.vue]: What is the purpose of the `<b>` tag here?

File: components/Article.vue [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+		</h1>
+		<p id="api-content" v-html="Content" />
```
Comment (greatislander) [inline components/Article.vue]: Is this the best element to contain the content of a given page?

File: components/Article.vue [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+<script>
+export default {
+	props: {
+		Title: {
+			type: String,
+			default: "Untitled Post"
+		},
+		Content: {
+			type: String,
+			default: "We Count content"
```
Comment (greatislander) [inline components/Article.vue]: What would be a better option for communicating that a page or post has no content?

File: components/Article.vue [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+		<p id="api-content" v-html="Content" />
+	</div>
+</template>
+
+<script>
+export default {
+	props: {
+		Title: {
+			type: String,
+			default: "Untitled Post"
```
Comment (greatislander) [inline components/Article.vue]: This component will be used for posts and pages. Can we make this more generic?

**Comment by jobara** [comment]
@TeddTech this [reference of HTML elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element) may be helpful.

**Pull request merged**